DigitalLearningSolutions.Data.Tests/Services/CourseCompletionServiceTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System;
+    using DigitalLearningSolutions.Data.Models.CourseCompletion;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    internal class CourseCompletionServiceTests
+    {
+        private CourseCompletionService courseCompletionService;
+
+        [SetUp]
+        public void Setup()
+        {
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            courseCompletionService = new CourseCompletionService(connection);
+        }
+
+        [Test]
+        public void Get_course_completion_of_assessed_course_should_return_course_completion()
+        {
+            // Given
+            const int candidateId = 252646;
+            const int customisationId = 21452;
+
+            // When
+            var result = courseCompletionService.GetCourseCompletion(candidateId, customisationId);
+
+            // Then
+            result.Should().BeEquivalentTo(new CourseCompletion(
+                customisationId,
+                "TNA v0.1",
+                "RiO Mersey Care & NWBH",
+                false,
+                DateTime.Parse("2017-07-26 07:56:15.273"),
+                null,
+                1,
+                true,
+                80,
+                85,
+                100,
+                22,
+                13,
+                9,
+                13
+            ));
+        }
+
+        [Test]
+        public void Get_course_completion_of_course_should_when_all_tutorials_status_0()
+        {
+            // CustomisationTutorials.Status = 0 entries where CustomisationID = 26696

```

Comment:
What exactly does this comment mean? 🙂

Reply:
That's not very clear, sorry! Changed to a hopefully cleared comment in 2cbdd33 to:

`CustomisationTutorials.Status = 0 for all CustomisationTutorial records where CustomisationID = 26696`